definitions/ads_domain/google_ads/update/update_fact_ad_performance_daily.sqlx

```diff
@@ -64,6 +64,24 @@ warehouse AS (
     clicks,
     cost
   FROM ${ref("fact_ad_performance_daily")}
+),
+landing_dedupt AS (
+  SELECT
+    ad_id,
+    date_id,
+    device,
+    MAX(account_status) AS account_status,
```

Comment:
@kingman these columns with MAX() operators to deduplicate are slow changing dimensions, right?

Click, costs, impressions can be set to MAX... to get the latest performance values.
However, statuses we should get the current statuses (check this: https://cloud.google.com/bigquery/docs/google-ads-transfer#query_your_data)